src/components/Reports/DonationsReport/Table/DonationsReportTable.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Button,
+  Link,
+  Divider,
+  styled,
+  Table,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import EditIcon from '@material-ui/icons/Edit';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { useTranslation } from 'react-i18next';
+import { DataGrid, GridColDef, GridCellParams } from '@material-ui/data-grid';
+import { DateTime } from 'luxon';
+import { EmptyDonationsTable } from '../../../common/EmptyDonationsTable/EmptyDonationsTable';
+
+interface Donation {
+  date: Date;
+  partnerId: string;
+  partner: string;
+  currency: string;
+  foreignCurrency: string;
+  convertedAmount: number;
+  foreignAmount: number;
+  designation: string;
+  method: string;
+  id: string;
+}
+
+interface Props {
+  data: Donation[];
+  accountListId: string;
+}
+
+const DataTable = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  '& .MuiDataGrid-row.Mui-even:not(:hover)': {
+    backgroundColor:
+      theme.palette.type === 'light'
+        ? theme.palette.common.white
+        : theme.palette.cruGrayLight.main,
+  },
+}));
+export const DonationsReportTable: React.FC<Props> = ({
+  data,
+  accountListId,
+}) => {
+  const { t } = useTranslation();
+  const accountCurrency = data[0].currency;
+  const link = (params: GridCellParams) => {
+    const row = params.row as Donation;
+    return (
+      <Typography>
+        <Link href={`../../${accountListId}/contacts/${row.partnerId}`}>
+          {params.value}
+        </Link>
+      </Typography>
+    );
+  };
+  const amount = (params: GridCellParams) => {
```

Comment:
Could you add spaces between these methods? It makes it a little bit easier to read.